### PR TITLE
Temporarily `xfail` `test_solve_sym_pos`

### PR DESCRIPTION
**Pull request by pavithraes**
Temporarily fix CI, ref #9335

### CODE REVIEW

File: dask/array/tests/test_linalg.py [hunk 1]
```diff
@@ -760,6 +760,7 @@ def _get_symmat(size):
     return lA.dot(lA.T)
 
 
+@pytest.mark.xfail(reason="https://github.com/dask/dask/issues/9335", strict=False)
```
Comment (jrbourbeau) [inline dask/array/tests/test_linalg.py:763]: Can we make it so this only is expected to fail when the appropriate version of `scipy` is installed? I think https://docs.pytest.org/en/7.1.x/how-to/skipping.html#condition-parameter is what we're after for that

**Review by jrbourbeau — COMMENTED** [review]
Thanks @pavithraes!

File: dask/array/tests/test_linalg.py (hunk 1 above)
Comment (pavithraes) [reply dask/array/tests/test_linalg.py]: Thanks, yes!

**Review by jrbourbeau — APPROVED** [review]
Thanks @pavithraes! Will merge after CI finishes. Note the unrelated `distributed` test failures are being handled over in https://github.com/dask/dask/pull/9337

**Pull request merged**